packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -40,6 +42,15 @@ class ParameterMetadata(NamedTuple):
     Required when 'type' is Kind.TypeMessage. Ignored for any other 'type'.
     """
 
+    def sanitized_display_name(self) -> str:
```

Comment:
"sanitized_display_name" is not wrong, but something like "field_name" or "protobuf_name" would better describe what this is for.